code/modules/power/apc/apc_main.dm

```diff
@@ -163,16 +163,16 @@
 	switch(dir)
 		if(NORTH)
 			offset_old = pixel_z
-			pixel_z = APC_PIXEL_OFFSET
+			pixel_z = 35
 		if(SOUTH)
 			offset_old = pixel_z
 			pixel_z = -APC_PIXEL_OFFSET
 		if(EAST)
 			offset_old = pixel_x
-			pixel_x = APC_PIXEL_OFFSET
+			pixel_x = 11
 		if(WEST)
 			offset_old = pixel_x
-			pixel_x = -APC_PIXEL_OFFSET
+			pixel_x = -11
```

Comment:
this code block shouldn't exist. you should override wall_mount_offset instead (on that topic this may fuck up display visuals depending on how pixel_z shakes out, I have a feeling the -APC_PIXEL_OFFSET bit is gonna look wrong)

Reply:
(oh also the dir set that occurs in init should be using setDir(), just as a rule

Reply:
this code still needs to be removed

Reply:
You right, got lost in the chaos